apps/site/assets/ts/stop/components/sidebar/Accessibility.tsx

```diff
@@ -91,9 +91,13 @@ const renderFeaturesList = (stop: Stop): ReactElement<HTMLElement> | null =>
       {isSpecialCase(stop) ? `${stop.name} ` : "It "}
       has the following features:
       <ul className="m-stop-page__sidebar-list">
-        {stop.accessibility.map(access => (
-          <Access access={access} key={access} />
-        ))}
+        {stop.name === "Lynn"
+          ? [stop.accessibility[0], stop.accessibility[2]].map(access => (
```

Comment:
I think this shouldn't rely on the positioning - what if the order of the strings returned changes?
We can filter by looking at the value. Something more like `stop.accessibilty.filter(access => access !== "elevator").map( // and so on`